### ISSUE TITLE
Sample Training set and Test set for the code

### ISSUE THREAD

**Issue by nitishkannan23** [Drusinek1/Well_Log_Classification#1]
Hi,

Can you please share a sample training set and test set like the one you used for your code.

Thanks!
